lib/pbench/server/api/resources/users_api.py

```diff
@@ -35,10 +38,7 @@ def post(self):
 
         :return:
             Success: 200 with password reset link sent to email
-            Failure: <status_Code>,
-                    response_object = {
-                        "message": "failure message"
-                    }
+            Failure: Success with a logger warning
```

Comment:
It would be good if these two were clearer, or at least more consistent with each other.  And, I think we generally prefer symbolic references to literals.  So, something like this would be better:
```
        :return:
            Success: OK status with password reset link sent to email
            Failure: OK status, with a logger warning -- no email sent
```